apps/web/modules/survey/follow-ups/lib/follow-ups.ts

```diff
@@ -0,0 +1,269 @@
+import { getOrganization } from "@/lib/organization/service";
+import { getResponse } from "@/lib/response/service";
+import { getSurvey } from "@/lib/survey/service";
+import { rateLimit } from "@/lib/utils/rate-limit";
+import { validateInputs } from "@/lib/utils/validate";
+import { sendFollowUpEmail } from "@/modules/email";
+import { getSurveyFollowUpsPermission } from "@/modules/survey/follow-ups/lib/utils";
+import { FollowUpResult, FollowUpSendError } from "@/modules/survey/follow-ups/types/follow-up";
+import { z } from "zod";
+import { TSurveyFollowUpAction } from "@formbricks/database/types/survey-follow-up";
+import { logger } from "@formbricks/logger";
+import { ZId } from "@formbricks/types/common";
+import { Result, err } from "@formbricks/types/error-handlers";
+import { TOrganization } from "@formbricks/types/organizations";
+import { TResponse } from "@formbricks/types/responses";
+import { TSurvey } from "@formbricks/types/surveys/types";
+
+const limiter = rateLimit({
+  interval: 60 * 60, // 1 hour
+  allowedPerInterval: 50, // max 50 calls per org per hour
+});
+
+export const evaluateFollowUp = async (
```

Comment:
Can we avoid exporting this method so the only entrypoint to sending followups is `sendSurveyFollowUps`?